test/libsolidity/semanticTests/abiEncoderV2/abi_decode_fixed_arrays.sol

```diff
@@ -1,12 +0,0 @@
-pragma experimental ABIEncoderV2;
-
```

Comment:
Why was this test removed?

Reply:
Ah nevermind this

Reply:
Has the same test in abiEncoderV1 without option ABIEncoderV1Only, so it will run for both encoders